product_supplierinfo_archive/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+Simply use the archive action on any product supplier info.
```

Comment:
Where the archive button is defined ?

Reply:
You can archive one product.supplierinfo by selecting one one the vendor pricelists view, and then click on action and archive.

Reply:
The action is created automatically if the field exists? I didn't know.
Well : nice to have, having a button on the product form view that disable a supplierinfo. (With a confirm message)
It could reduce the number of click to archive a supplierinfo, dont' you think ?

Reply:
Thanks for the idea, here's how it looks.
![product_supplierinfo_archive](https://user-images.githubusercontent.com/91118904/147821256-ccc45a99-4d89-4be4-9036-c12b7f6f859d.gif)
You tell me what you think about it! :smile: 
@OriolVForgeFlow